# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/mkdosfs/dont-use-kernel-headers.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./mkdosfs.c.orig	2004-02-10 22:44:38.602762200 +0000
+++ ./mkdosfs.c	2004-02-10 22:43:57.846958024 +0000
@@ -43,7 +43,7 @@
 
 #include <fcntl.h>
 #include <linux/hdreg.h>
-#include <linux/fs.h>
+//#include <linux/fs.h>
 #include <sys/stat.h>
 #include <linux/fd.h>
 #include <mntent.h>
@@ -61,6 +61,9 @@
 
 /* Constant definitions */
 
+#define BLOCK_SIZE_BITS 10
+#define BLOCK_SIZE (1<<BLOCK_SIZE_BITS)
+
 #define TRUE 1			/* Boolean constants */
 #define FALSE 0
 
